diff -up kdesdk-kioslaves-15.08.3/CMakeLists.txt.svn_optional kdesdk-kioslaves-15.08.3/CMakeLists.txt
--- kdesdk-kioslaves-15.08.3/CMakeLists.txt.svn_optional	2015-11-04 07:06:54.000000000 -0600
+++ kdesdk-kioslaves-15.08.3/CMakeLists.txt	2015-11-30 09:15:15.679119551 -0600
@@ -15,7 +15,7 @@ macro_log_feature(SVN_FOUND "SVN" "Subve
 macro_log_feature(PERL_FOUND "Perl" "Perl Programming Language" "http://www.perl.org/" FALSE "5.10" "Required in order to use the perldoc KIOSlave.")
 
 if(SVN_FOUND)
-  add_subdirectory(svn)
+  macro_optional_add_subdirectory(svn)
 endif(SVN_FOUND)
 
 if(PERL_FOUND)
